### PR TITLE
Add styles to components

### PR DESCRIPTION
**Pull request by Ellon-M**
## Changes 

- Wrote CSS to style the different components on the page.

### CODE REVIEW

File: src/components/AddBook.js [hunk 1]
```diff
@@ -30,19 +30,20 @@ const AddBook = () => {
...
+
+        <select className="py-3 px-5 outline-none cursor-pointer bg-white border rounded-md border-[#a0a0a0]" name="category" onChange={(e) => setCategory(e.target.value)}>
           {categories && categories.map((category) => (
             <option value={category} key={category}>{category}</option>
           ))}
         </select>
-        <button type="submit" className="bg-gray-100 ml-5 p-2">Add Book</button>
+
+        <button type="submit" className="bg-[#0290ff] ml-2 py-2 px-5 text-white font-semibold">Add Book</button>
       </form>
```
Comment (AmaduKamara) [inline src/components/AddBook.js]: - [ ] The form input fields and the Add Book button should span across the full width left - right of the parent container in order to match the original design. Kindly revisit the original design and style your app exactly similarly to that design. 👍🏽

**Your Design**

Kindly work on the form as it points at the bottom of the screenshot to match your design with the original design given 👍🏽

Screenshot 2023-01-27 at 3 50 06 PM

**The Original Design**

Screenshot 2023-01-27 at 3 42 37 PM

File: src/components/Book.js [hunk 2]
```diff
@@ -12,11 +12,16 @@ const Book = ({
   };
 
   return (
-    <div className="my-5">
+    <div className="my-5 border-b">
```
Comment (AmaduKamara) [inline src/components/Book.js]: - [ ] Each book in the book list is styled with a border around it and not only the bottom. Having a border right around, a border radius, padding and a little bit of shadow gives each book an elegant look. You could replace this div's styles with this ` <div className="my-5 border p-3 rounded-md">` and you will see how nice it looks. 👍🏽
- [ ] A single book is expected to have the percentage section and the UPDATE PROGRESS button. They can just be static designs with no functionality. Later in your free time when you would want to master react, you could work on them but for this project, you're asked to only have the design look exactly and not to work on their functionality. Please add those sections, the middle percentage and the last one with the update progress button just as in the original design 👍🏽


Kindly look at the screenshot below and see the areas marked in green that you should consider working on.

**The Original Design**

Screenshot 2023-01-27 at 3 42 37 PM

**Your Design**

Screenshot 2023-01-27 at 3 50 06 PM

**Review by AmaduKamara — CHANGES_REQUESTED** [review]
# Status: Required Changes ♻️ ♻️ ♻️

Hi @Ellon-M,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✅ You used Gitflow correctly 👍🏽
✅ Linter checks passed 👍🏽
✅ The app works fine as expected 👍🏽

## Required Changes ♻️
_Check the comments under the review._

- [ ] It would be nice if you could add the user icon on the navbar so your design looks just like the original design given and as well improve the look and feel of your app 👍🏽
- [ ] Kindly work on the form styles to match your design with the original given design 👍🏽
- [ ] A single book is expected to look like a card. Please improve their styles to match the given design. 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by sumairq — APPROVED** [review]
### Hi @Ellon-M , 



Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Good commit history
✔️Linters are passing
✔️Very professional README
✔️Correct Git flow
✔️Descriptive PR
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**